assets/src/components/app.tsx

```diff
@@ -1,78 +1,95 @@
-import React from "react"
+import React, { ReactElement } from "react"
+import { BrowserRouter, NavLink, Route as BrowserRoute } from "react-router-dom"
 import {
   Route,
   RouteId,
   TimepointsByRouteId,
-  Vehicle,
   VehicleId,
   VehiclesByRouteId,
 } from "../skate"
-import RouteLadders from "./routeLadders"
-import RoutePicker from "./routePicker"
-import VehiclePropertiesPanel from "./vehiclePropertiesPanel"
+import FAQPage from "./faqPage"
+import RoutePage from "./routePage"
 
 interface Props {
+  routePickerIsVisible: boolean
   routes: Route[] | null
   timepointsByRouteId: TimepointsByRouteId
   selectedRouteIds: RouteId[]
   vehiclesByRouteId: VehiclesByRouteId
   selectedVehicleId: VehicleId | undefined
 }
 
-export const findRouteById = (
-  routes: Route[] | null,
-  routeId: RouteId
-): Route | undefined => (routes || []).find(route => route.id === routeId)
-
-const findSelectedVehicle = (
-  vehiclesByRouteId: VehiclesByRouteId,
-  selectedVehicleId: VehicleId | undefined
-): Vehicle | undefined =>
-  Object.values(vehiclesByRouteId)
-    .reduce((acc, vehicles) => acc.concat(vehicles), [])
-    .find(vehicle => vehicle.id === selectedVehicleId)
-
-const vehicleRoute = (
-  allRoutes: Route[] | null,
-  vehicle: Vehicle | undefined
-): Route | undefined =>
-  (allRoutes || []).find(route => route.id === (vehicle && vehicle.routeId))
-
-const App = ({
-  routes,
-  timepointsByRouteId,
-  selectedRouteIds,
-  vehiclesByRouteId,
-  selectedVehicleId,
-}: Props) => {
-  const selectedRoutes: Route[] = selectedRouteIds
-    .map(routeId => findRouteById(routes, routeId))
-    .filter(route => route) as Route[]
-
-  const selectedVehicle = findSelectedVehicle(
-    vehiclesByRouteId,
-    selectedVehicleId
-  )
-
-  return (
+const App = (props: Props): ReactElement<HTMLDivElement> => (
+  <BrowserRouter>
     <div className="m-app">
-      <RoutePicker routes={routes} selectedRouteIds={selectedRouteIds} />
-
-      <RouteLadders
-        routes={selectedRoutes}
-        timepointsByRouteId={timepointsByRouteId}
-        vehiclesByRouteId={vehiclesByRouteId}
-        selectedVehicleId={selectedVehicleId}
-      />
+      <TabBar {...props} />
```

Comment:
The `TabBar` only uses one field (`routePickerIsVisible`). I think it's a little cleaner to not pass arguments that won't be used. (Especially if `TabBar` ends up getting its own file.)